src/site/stages/build/process-cms-exports/index.js

```diff
@@ -218,7 +218,7 @@ const entityAssemblerFactory = contentDir => {
 
     // Handle circular references
     const a = findCircularReference(entity, ancestors);
-    if (a) return a;
+    if (a) return a.entity;
```

Comment:
Is this change also related the output schema validation?

Reply:
I noticed it wasn't part of https://github.com/department-of-veterans-affairs/vets-website/pull/13700.

Reply:
No, there was a problem in which the child was returned wrapped into an entity object and wan't able to transform the data. This removes the entity wrap and returns a regular entity.